.github/workflows/tests.yml

```diff
@@ -36,6 +36,7 @@ jobs:
       E2E_TEST_AEST_OPERATIONS_USER: ministry-user-aest-operations@e2e-tests
       E2E_TEST_AEST_OPERATIONS_ADMINISTRATORS_USER: ministry-user-aest-operations-administrators@e2e-tests
       E2E_TEST_AEST_MOF_OPERATIONS_USER: ministry-user-mof-operations@e2e-tests
+      E2E_TEST_AEST_READ_ONLY_USER: ministry-user-read-only@e2e-tests
```

Comment:
Can we have it added to the sample file? The user names can be part of the sample also.
Can you please update the other E2E test users there? I completely forgot during the previous PRs.
@ann-aot @andrepestana-aot @sh16011993 please update your local env files with this also.